### ISSUE TITLE
Would it make sense to publish this as a nuget packege on nuget.org

### ISSUE THREAD

**Issue by gyurisc** [fretje/MembershipStarterKit#2]
I was wondering if it would make sense to package this project as a nuget package and publish it on nuget.org? That would make it very easy to add this functionality to your asp.net mvc project.

**Comment by fretje** [comment]
Of course, that would make a lot of sense! It is something that is definitely on my todo list, but I haven't yet found the time to do it though. You're more than welcome to give it a try ;-)

**Comment by gyurisc** [comment]
I would love to try to do it. I think the first thing step to take is to fork, then implement the changes and somehow then submit it back to your repository. Is it correct? I am new to github, that is why I am asking dumb questions :)

On the other hand what are the manual steps to add your module to my MVC project? Is there anything else besides referencing the dll?

**Comment by fretje** [comment]
Yes indeed, I'm fairly new to github as well, but you indeed have to fork, then make your changes, and then issue a pull request to me, so I can pull your changes into my repository. For instance, there's [a pending pull request](https://github.com/fretje/MembershipStarterKit/pull/1) from [JamesKovacs](https://github.com/JamesKovacs) right now.
Otherwise let me first pull his changes into my repository, so you can then start from there for creating a nuget package.

**Comment by fretje** [comment]
I just merged that pull request. It is now also possible to create new users. Btw, this should also fix [this issue](https://github.com/fretje/MembershipStarterKit/issues/3), as everything is/should be referenced properly now.

**Comment by fretje** [comment]
About your question " what are the manual steps to add your module to my MVC project?" Everything you need to know to use the dll should be in [README.markdown](https://github.com/fretje/MembershipStarterKit/blob/master/README.markdown). Take a look, and let me know if it isn't clear enough, or if something should be added.

**Comment by gyurisc** [comment]
I finally managed to read the readme and everything is clear now! In fact, I managed to run the sample project at try it out
